frontend/src/components/DropboxChooser/index.tsx

```diff
@@ -60,9 +65,10 @@ const DropboxChooser: FC<Props> = ({ children, onUploadFile }) => {
 
   return (
     <>
-      <div onMouseOver={handleMouseOver} onClick={handleClick}>
-        {children}
-      </div>
+      {React.cloneElement(children, {
```

Comment:
`DropboxChooser` needs to clone single child and pass `onClick` and `onMouseOver` instead of using a wrapper div to handle events, because this is more semantically correct than before (the button itself handles the events)